listing.html

```diff
@@ -0,0 +1,224 @@
+
+
+        <!--5th-->
+        <div class = "col-md-6 col-lg-4 col-xl-3 p-2 new">
+            <div class = "collection-img position-relative">
+                <img src = "images/FifthHouse.jpg" class = "w-100">
+                <span class = "position-absolute bg-danger text-white d-flex align-items-center justify-content-center">On Sale</span>
+            </div>
+            <div class = "text-center">
+                <p class = "text-capitalize my-1">FifthHouse</p>
+                <span class = "fw-bold" style=color:red>$750,000 </span>
+            </div>
+        </div>
+
+        <!--6th-->
+        <div class = "col-md-6 col-lg-4 col-xl-3 p-2 new">
+            <div class = "collection-img position-relative">
+                <img src = "images/SixthHouse.jpg" class = "w-100">
+                <span class = "position-absolute bg-danger text-white d-flex align-items-center justify-content-center">On Sale</span>
+            </div>
+            <div class = "text-center">
+                <p class = "text-capitalize my-1">SixthHouse</p>
+                <span class = "fw-bold" style=color:red>$800,000 </span>
+            </div>
+        </div>
+
+        <!--7th-->
+        <div class = "col-md-6 col-lg-4 col-xl-3 p-2 new">
+            <div class = "collection-img position-relative">
+                <img src = "images/SeventhHouse.jpg" class = "w-100">
+                <span class = "position-absolute bg-danger text-white d-flex align-items-center justify-content-center">On Sale</span>
+            </div>
+            <div class = "text-center">
+                <p class = "text-capitalize my-1">SeventhHouse</p>
+                <span class = "fw-bold" style=color:red>$900,000 </span>
+            </div>
+        </div>
+
+        <!--Footer-->
+        <nav class="bg-dark navbar mt-5 navbar-expand-md navbar-dark">
+            <div class="container-fluid">        
+                <a href="#" class="navbar-brand"></a>
+                <ul class="navbar-nav mt-auto">
+                    <li class="nav-item">
+                    <a href="#" class="nav-link">Help</a>
+                    </li>
+                    <li class="nav-item">
+                    <a href="#" class="nav-link">About Us</a>
+                    </li>
+                    <li class="nav-item">
+                    <a href="#" class="nav-link">Contact Us</a>
+                    </li>
+                    <li class="nav-item">
+                    <a href="#" class="nav-link">Terms of Use</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+
+
+    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script async
+    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB5AiQSg_qzK6Tcc381RnYIehBoubiaYaQ&callback=initMap">
+</script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="app.js"></script>
```

Comment:
embed the content in app.js here for now to reduce number of files because we will not use google map later on